stdlib/public/core/Collection.swift

```diff
@@ -782,6 +776,9 @@ public protocol Collection: Sequence where SubSequence: Collection {
   /// - Parameter i: A valid index of the collection. `i` must be less than
   ///   `endIndex`.
   func formIndex(after i: inout Index)
+
+  @available(swift, deprecated, message: "all index distances are now of type Int")
+  typealias IndexDistance = Int
```

Comment:
`public`.

Reply:
Yup. I moved it back and forth between the protocol and the extension multiple times and forgot this every. single. time.